packages/custom-elements-scoped/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@webcomponents/custom-elements-scoped",
```

Comment:
Maybe `scoped-custom-elements`? (For some reason `custom-elements-scoped` to me kinda sounds like it's a variant of the full custom-elements polyfill that also supports scoped). But I don't have a strong opinion.

Reply:
👍 Yeah, I take your point. I'm not sure flipping it helps a ton with that particular confusion (i.e. do I need both `custom-elements` and `scoped-custom-elements`?). But at a minimum it reads less awkward that way.

Reply:
Should we have registry in the name?

`@webcomponents/scoped-registry`
`@webcomponents/scoped-elements-registry`
`@webcomponents/scoped-custom-elements-registry`

Reply:
Changed to `scoped-custom-elements-registry`